src/Discord.Net.Core/Entities/Channels/TextChannelProperties.cs

```diff
@@ -9,18 +11,29 @@ public class TextChannelProperties : GuildChannelProperties
         /// <summary>
         ///     Gets or sets the topic of the channel.
         /// </summary>
+        /// <remarks>
+        ///     Setting this value to any string other than <c>null</c> or <see cref="string.Empty"/> will set the
+        ///     channel topic or description to the desired value.
+        /// </remarks>
         public Optional<string> Topic { get; set; }
         /// <summary>
         ///     Gets or sets whether this channel should be flagged as NSFW.
         /// </summary>
+        /// <remarks>
+        ///     Setting this value to <c>true</c> will mark the channel as NSFW (Not Safe For Work) and will prompt the
+        ///     user about its maturity nature before they may view the channel; setting this value to <c>false</c> will
+        ///     mark this channel as SFW (Safe For Work).
```

Comment:
Might be more accurate to say:
```suggestion
        ///     remove this NSFW indication.
```